packages/tests/playwright.config.ts

```diff
@@ -4,7 +4,7 @@ import { config as envConfig } from "./config";
 
 function createPlaywrightConfig(): PlaywrightTestConfig {
   return {
-    timeout: 16 * 1000,
+    timeout: 45 * 1000,
```

Comment:
Is this related to the timeout increase? If yes, then I'd mock timers instead of increasing the timeout 🤔

Reply:
Of course. Problem is, it would blow up the scope of this work. These are not unit tests so we don't have that level of control.

E2E tests test apps which depend on the React SDK. Multiple boundaries are in place so we would need a system for injecting a test config when tests are being run. This would likely take a lot of time so I went with the easy way this time.

Reply:
Understood, we should at least make a note of this because it directly makes the CI slower 🥶